src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="main">
+      <div class="main__card card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="#"
+          class="card__img"
+        >
+        <div class="card__content">
+          <p class="card__text">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </p>
+          <p class="card__code">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__rating-p">Отзывов: 5</p>
```

Comment:
думаю не стоит писать название тега в классе
```suggestion
            <p class="card__rating-value">Отзывов: 5</p>
```